.github/workflows/ubuntu-21.04.yml

```diff
@@ -0,0 +1,55 @@
+name: Ubuntu 21.04 CI
+
+on: [push, workflow_dispatch]
+
+jobs:
+  build-devenv:
+    runs-on: ubuntu-20.04
```

Comment:
20 or 21? :)

Reply:
Yeah it's a bit tricky here. This workflow runs on 20.04 machines (provided by GitHub, see https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners), but the build environment is a docker image built from Ubuntu 21.04 (because 21.04 ships gcc-11).